ipyleaflet/leaflet.py

```diff
@@ -207,6 +207,21 @@ class VideoOverlay(RasterLayer):
     attribution = Unicode().tag(sync=True, o=True)
 
 
+class Heatmap(RasterLayer):
+    _view_name = Unicode('LeafletHeatmapView').tag(sync=True)
+    _model_name = Unicode('LeafletHeatmapModel').tag(sync=True)
+
+    latlngs = List().tag(sync=True)
+
+    # Options
+    minOpacity = Float(0.05).tag(sync=True, o=True)
+    maxZoom = Int(18).tag(sync=True, o=True)
```

Comment:
`ipyleaflet` automatically converts `snake_case` to `camelCase` between the front-end and the back-end, so normally you should be able to call these traits `min_opacity` and `max_zoom` without any other change.

Reply:
Yes I discovered that recently. Any reason we have `snake_case` on the Python side and `camelCase` on the JS side, and not the same names on both sides?

Reply:
Mostly because of PEP8.